modules/ROOT/nav.adoc

```diff
@@ -34,3 +34,7 @@
 ** xref:migrate-cl.adoc[Migrating from Container Linux]
 * Reference pages
 ** xref:platforms.adoc[Platforms]
+* xref:tutorials.adoc[Tutorials]
+** xref:getting-started-tutorial.adoc[Getting Started]
```

Comment:
We already have a separate "Getting Started" page. To avoid confusion, let's call this one something like "Basic Provisioning and Customization".

Also, I'd be happier if we could name all the files `tutorial-<FOO>.adoc`.

Reply:
Renamed the tutorial files to: `tutorial-<FOO>.adoc`, https://github.com/coreos/fedora-coreos-docs/pull/134/commits/b8c49f9570196ebbed0ac8c7c866b9e7f17d8879